targets/unity-v2/source/Shared/Internal/PlayFabHttp/PlayFabUnityHttp.cs

```diff
@@ -55,30 +60,36 @@ private static IEnumerator SimpleCallCoroutine(string fullUrl, byte[] payload, A
             }
             else
             {
-                var putRequest = UnityWebRequest.Put(fullUrl, payload);
-#if UNITY_2017_2_OR_NEWER
-                putRequest.chunkedTransfer = false; // can be removed after Unity's PUT will be more stable
-                putRequest.SendWebRequest();
-#else
-                putRequest.Send();
-#endif
 
-#if !UNITY_WEBGL
-                while (putRequest.uploadProgress < 1 && putRequest.downloadProgress < 1)
+                UnityWebRequest request;
+                if (method == "put")
                 {
-                    yield return 1;
+                    request = UnityWebRequest.Put(fullUrl, payload);
                 }
-#else
-                while (!putRequest.isDone)
+                else
                 {
-                    yield return 1;
+                    request = new UnityWebRequest(fullUrl, "POST");
```

Comment:
You are converting "GET" calls to "POST" which will make all GET calls fail.
(You're not handling get separately from post)